package.json

```diff
@@ -6,9 +6,20 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "axios": "^0.19.2",
+    "history": "^4.10.1",
+    "lodash": "^4.17.15",
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
-    "react-scripts": "3.4.0"
+    "react-redux": "^7.2.0",
+    "react-router": "^5.1.2",
```

Comment:
Another suggestion, consider starting with `@reach/router`, it will eventually be merged into react-router but the APIs are going to be more similar to what looks like in `@reach/router`, essentially they want to do a lot of things using Hooks whenever possible. 

More context here: https://reacttraining.com/blog/reach-react-router-future/ 

😄